expo/gbe/views/create_volunteer_view.py

```diff
@@ -54,6 +95,10 @@ def CreateVolunteerView(request):
                 for window in form.cleaned_data['unavailable_windows']:
                     volunteer.unavailable_windows.add(window)
 
+                for interest_form in formset:
+                    vol_interest = interest_form.save(commit=False)
```

Comment:
Reviewing the code, I see that we can't get here with a bad form in the formset. But the fact that this was hard does indicate that readability is an issue here. Extraction and naming of the offending code is fine, we'll fix it next time it gets touched.